q2_demux/tests/test_demux.py

```diff
@@ -787,20 +787,25 @@ def test_inconsistent_sequence_length_single(self):
 
         demux_data = emp_single(bsi, barcode_map)
         with tempfile.TemporaryDirectory() as output_dir:
-            with self.assertRaisesRegex(ValueError,
-                                        'Observed sequences of length'):
-                # TODO: Remove _PlotQualView wrapper
-                summarize(output_dir, _PlotQualView(demux_data,
-                                                    paired=False), n=2)
+            # TODO: Remove _PlotQualView wrapper
```

Comment:
Never mind, I think I understand what you have in mind and am almost done writing some tests which iterate over that.  Thanks!